### DISCUSSION TITLE
Detailing my last two weeks trying and failing to get a Stripe Express account setup

### DISCUSSION THREAD

**Discussion by endigo9740** [community/community#40774] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, if anyone can see this, I'd like to share my experience getting my new business and organization setup to use GitHub Sponsors for our open source project at:

https://github.com/skeletonlabs/skeleton

More specifically share our experience setting up the Stripe Express account required for payment processing. Here's the overall timeline of events:

- Nov 15 (2 weeks ago): We signed up for GitHub Sponsors
- Nov 23 (11 days ago): Reach out to Stripe support to report an issue uploading our EIN SS-4 document
- Nov 27 (7 days ago): Finally able to reach Stripe phone support, 2nd screenshot email thread starts below
- Nov 28 (6 days ago): Find out there's issues with new EINs, but they grant a 60 days grace period, this does not help
- Nov 29 (5 days ago): The last communication from Stripe support, they say an internal team is reviewing our issue

Here's the full email communication threads with private info removed:

![stripe-emails-1](https://user-images.githubusercontent.com/1509726/205510235-e58369bf-4804-4d3f-bc47-11ebff15bd87.jpeg)

![stripe-emails-2](https://user-images.githubusercontent.com/1509726/205510237-174a5d27-0c19-43b2-9adf-47c10596555c.jpeg)

To summarize - we've been communicating with support for two weeks on this issue, have clearly explained our issue and provided screenshots, Stripe support has provided written assurance almost a week ago that we're verified and can begin to use this service, but their dashboard is still in an error state. This is blocking us from using their service, which is blocking us from using GitHub Sponsors. I do not have confidence this issue will ever be resolved, meaning we are essentially blocked from using GitHub Sponsors.

I have to wonder, is this the level of quality you expect for this program? No other alternative payment processors are available so this could be the experience for any new business. Aside from the issue with the new EIN (which is a simple mistake on our part), we've done everything asked of us, yet we still cannot use Stripe Express and thus cannot use GitHub Sponsors. We're looking into alternative sponsor services, but otherwise I don't know what else to do here.

Open source lives and dies based on funding. GitHub Sponsors is supposed to be the official GitHub solution for this. Is there anyone on your side that is willing to help communicate with Stripe and help us out? Stripe support seemingly intentionally blocks phone and live chat service for users (we're always redirected to email), which means we have zero lines of communication with them aside from sending email messages that are promptly ignored.

Please, I'm begging you, can anyone from the GitHub Sponsors or Support teams help us out?

**Comment by endigo9740** [comment]
Just to add to this. I signed up for both Patreon and Ko-Fi today, both of which use Stripe (not Stripe Express), and ZERO issues. Had my accounts setup within an afternoon. Just to put this into perspective.

**Reply by seeuon19** [reply]
Sadly I've had an almost identical experience trying to get Stripe to verify my EIN. LIke you I'm not confident it will ever get resolved and am feeling very frustrated that I can reach anyone who can solve the problem. In my case all of my documents match the data I entered but still their system is not not accepting it. In my case I'm trying to setup GiveButter for my nonprofit and I've been told there is nothing they can do. If your Stripe situation finally got resolved I would love to hear how long it took.

**Reply by endigo9740** [reply]
@seeuon19 it ended up tacking some back and forth with a GitHub support rep, but we end up identifying the issue being that the EIN was too new and not yet available to use. In our case, this was a brand new company, so the EIN had just been issued. It took upwards of a month+ for this to show up in the government database so that Stripe could utilize this.

**Reply by seeuon19** [reply]
I see. This is probably the case in our situation as well because we just started a new nonprofit on 11/15/23. But curious why the experience seemed to be different when using Stripe Express vs Stripe?

**Reply by endigo9740** [reply]
No clue. My guess is Express actually tests the EIN upfront. And there was no way to check the status of the EIN, so it was just a matter of testing over and over until it took. But I think it was around a month if I recall from the time it arrived to the time it could be used.

I feel for you, it's a rough situation!

**Reply by seeuon19** [reply]
Got it. Thank you. Finding this thread really helped. At least now I don't feel so crazy and can now see the light at the end of the tunnel. Happy Holidays!

**Comment by ossanna16** [comment]
Hi there @endigo9740 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
